packages/components/src/DateTimePickers/LegacyDateTimePickers/DateTime/date-extraction.js

```diff
@@ -147,7 +152,12 @@ function checkMinutes(minutes) {
 	const minsNum = Number(minutes);
 	if (minutes === '') {
 		return new DatePickerException('INVALID_MINUTES', 'INVALID_MINUTES_EMPTY');
-	} else if (minutes.length !== 2 || isNaN(minsNum) || minsNum < 0 || minsNum > 59) {
+	} else if (
+		(minutes.length > 0 && minutes.length !== 2) ||
```

Comment:
minutes.length > 0 is not necessary no ? minutes is a string, minsNum is the converted value on type number. minsNum < 0 is already done on L158